src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -420,5 +420,9 @@ public static string CertMustHavePrivateKey(string certificateName)
         public const string OboCacheKeyNotInCache = "The token cache does not contain a token with an OBO cache key that matches the longRunningProcessSessionKey passed into ILongRunningWebApi.AcquireTokenInLongRunningProcess method. Call ILongRunningWebApi.InitiateLongRunningProcessInWebApi method with this longRunningProcessSessionKey first or call ILongRunningWebApi.AcquireTokenInLongRunningProcess method with an already used longRunningProcessSessionKey. See https://aka.ms/msal-net-long-running-obo .";
 
         public const string MultiCloudSupportUnavailable = "Multi cloud support unavailable with broker.";
+
+        public const string RequestFailureErrorMessage = "=== Token Acquisition ({0}) failed.";

```

Comment:
why the "==="?
is it for formatting?

Reply:
Yes just for formatting. Actually copied the way it is logged in RequestBase.